kitsune/wiki/content_managers.py

```diff
@@ -261,18 +299,77 @@ def create_revision(
             data, document, Profile.get_sumo_bot(), based_on_id, base_rev, send_notifications
         )
 
-    def publish_revision(self, revision: Revision, user=None, send_notifications=True) -> Revision:
-        """Publish (approve) a revision using the sumo bot.
-
-        Args:
-            revision: The revision to publish
-            send_notifications: Whether to send content creation notifications
-        """
+    def publish_revision(
+        self, revision: Revision, user=None, comment=None, send_notifications=True
+    ) -> Revision:
+        """Publish (approve) a revision using the sumo bot."""
         user = user or Profile.get_sumo_bot()
-        return super().publish_revision(revision, user, send_notifications)
+        rev = super().publish_revision(revision, user, comment, send_notifications)
+        self.fire_notifications(rev, [NotificationType.REVIEW_WORKFLOW])
+        render_document_cascade.delay(rev.document.id)
+        return rev
 
 
 class HybridContentManager(AIContentManager):
     """Content manager for hybrid translation workflow."""
 
-    pass
+    def __init__(self):
+        unreviewed_translations = Revision.objects.filter(
```

Comment:
Up until now, the manager was responsible for the operations on a single revision and strategy was doing the orchestration. This change breaks this architecture and couples document discovery for translation approval with document management.  The manager could provide methods like `is_stale`, `can_be_approved` (names are just examples) which would be responsible for introspecting the single revision.
My suggestion would be to keep the separation of concerns and move the discovery of documents to the strategy and let the management of the document to the manager